### PR TITLE
#149 fix home console errors

### PR DESCRIPTION
**Pull request by mort3za**

### CODE REVIEW

**Comment by mort3za** [comment]
#149

**Comment by mort3za** [comment]
@atharva3010 merge this one please

File: src/components/Home.vue [hunk 1]
```diff
@@ -220,156 +220,11 @@ export default {
   },
   data() {
     return {
-      accommodations: {
```
Comment (atharva3010) [inline src/components/Home.vue]: why did you remove these? They are needed to not cause any 'length undefined' errors when we use them as a response from api.

Comment (atharva3010) [reply src/components/Home.vue]: we need the api response structure defined.

Comment (StrangeWill) [reply src/components/Home.vue]: I _think_ this is just a progression from a very early iteration of this page, it's odd because further down all API calls pull arrays from the category list, this always had a mismatching type since the category system was implemented (mumble grumble, typescript prevents this), which is probably part of the errors we got (also we have a double-slicing going on, which again is something we did on the listing page for display purposes but probably ended up over here at some point).

Yeah all objects should be defined for Vue to be reactive properly, but everything is an array as was dropped in as part of this PR, not these object trees.


If I had to wager a guess, it has something to do with this commit: https://github.com/whynotearth/hotel-website-vkirirom.com/blame/be3f1adc7696d4a828f98833ef5a4ff0679015a0/src/components/Home.vue when we _were_ pulling individual page records and binding them.

---

Basically at some point we stopped loading individual pages and started loading arrays of page information, the main data never got properly updated (or more accurately, removed), this remedies this.
